src/server/api/routers/subscription.test.ts

```
@@ -0,0 +1,321 @@
+
+describe("subscriptionRouter.getStatus", () => {
+  it("throws UNAUTHORIZED when not logged in", async () => {
+    const caller = makeAnonCaller();
+
+    await expect(caller.getStatus()).rejects.toMatchObject({ code: "UNAUTHORIZED" });
+  });
+
+  it("returns subscription when one exists", async () => {
+    const fakeSub = {
+      userId: "user-1",
+      isActive: true,
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'isActive' should be before 'userId'.

(sort-keys-fix/sort-keys-fix)